### ISSUE TITLE
Humble request from distro packagers to start releasing versions 

### ISSUE THREAD

**Issue by kasicka** [WebAssembly/wasi-libc#317]
Hi, me and @khardix are fedora/RHEL package maintainers and it would help us a big lot if you could start making releases granting some level of ABI/API compatibility.

wasi-libc is used by several projects (ie rust and nodejs) and our goal is having one library that could be used by all/most of them. Currently the projects rely on specific commits, which is not very universal, leading us to bundling, which is against policies.

**Comment by sbc100** [comment]
It seems reasonable that we could start tagging releases.   I imagine it would be sufficient to tag the versions that have gone into wasi-sdk releases, and to use the same versioning scheme: https://github.com/WebAssembly/wasi-sdk/releases.

We could even make the tags retrospectively which should allow those packages to point to a specific release.

**Comment by sbc100** [comment]
@sunfishcode looks like we started tagging releases at the same time :)   I'll delete the 3 tags I just made and leave it to you.  Maybe tag at least the last 3 releases?

For reference
16: 30094b6ed05f19cee102115215863d185f2db4f0
15: 9886d3d6200fcc3726329966860fc058707406cd
14: ad5133410f66b93a2381db5b542aad5e0964db96

**Comment by sunfishcode** [comment]
Makes sense to me too. I've now created a [wasi-sdk-16](https://github.com/WebAssembly/wasi-libc/releases/tag/wasi-sdk-16) tag of wasi-libc, which is the revision of wasi-libc in [wasi-sdk-16](https://github.com/WebAssembly/wasi-sdk/releases/tag/wasi-sdk-16).

@sbc100 Oops, looks like we raced :-}.

@kasicka We do work to avoid API and ABI breaks in general. That said, there are some major changes to WASI coming up which may require some incompatible changes over the coming months. We'll try to minimize the disruption, but it's likely that we'll have at least some ABI breaks. But hopefully with tagged versions, users will be able to manage when those changes show up.
